### ISSUE TITLE
Game doesn't load

### ISSUE THREAD

**Issue by Sketano** [Crendgrim/AutoHUD#72]
[latest.log](https://github.com/Crendgrim/AutoHUD/files/12817584/latest.log)
 Heres my latest.log, Not sure how Github works, so here it is In text:
 
```
[05Oct2023 14:58:36.426] [main/INFO] [cpw.mods.modlauncher.Launcher/MODLAUNCHER]: ModLauncher running: args [--username, _Sketano, --version, 1.20.1, --gameDir, C:\Users\chump\AppData\Roaming\com.modrinth.theseus\profiles\Revitalized, --assetsDir, C:\Users\chump\AppData\Roaming\com.modrinth.theseus\meta\assets, --assetIndex, 5, --uuid, 989d88f0eb7e436fb0dad72b97fa4079, --accessToken, ????????, --clientId, c4502edb-87c6-40cb-b595-64a280cf8906, --xuid, 0, --userType, msa, --versionType, release, --width, 854, --height, 480, --launchTarget, forgeclient, --fml.forgeVersion, 47.1.3, --fml.mcVersion, 1.20.1, --fml.forgeGroup, net.minecraftforge, --fml.mcpVersion, 20230612.114412]
[05Oct2023 14:58:36.449] [main/INFO] [cpw.mods.modlauncher.Launcher/MODLAUNCHER]: ModLauncher 10.0.9+10.0.9+main.dcd20f30 starting: java version 17.0.8 by Azul Systems, Inc.; OS Windows 11 arch amd64 version 10.0
[05Oct2023 14:58:39.990] [main/INFO] [net.minecraftforge.fml.loading.ImmediateWindowHandler/]: Loading ImmediateWindowProvider fmlearlywindow
[05Oct2023 14:58:40.286] [main/INFO] [EARLYDISPLAY/]: Trying GL version 4.6
[05Oct2023 14:58:40.751] [main/INFO] [EARLYDISPLAY/]: Requested GL version 4.6 got version 4.6
[05Oct2023 14:58:41.129] [pool-2-thread-1/INFO] [EARLYDISPLAY/]: GL info: AMD Radeon(TM) RX Vega 10 Graphics GL version 4.6.0 Core Profile Context 23.7.2.230718, ATI Technologies Inc.
[05Oct2023 14:58:42.846] [main/INFO] [gg.essential.loader.stage1.EssentialLoaderBase/]: Starting Essential Loader (stage2) version 1.4.1 (4b3b87be87d34916028dffa85460d085) [stable]
[05Oct2023 14:58:43.111] [main/INFO] [mixin/]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=union:/C:/Users/chump/AppData/Roaming/com.modrinth.theseus/meta/libraries/org/spongepowered/mixin/0.8.5/mixin-0.8.5.jar%23100!/ Service=ModLauncher Env=CLIENT
[05Oct2023 14:58:45.846] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModFileParser/LOADING]: Mod file C:\Users\chump\AppData\Roaming\com.modrinth.theseus\meta\libraries\net\minecraftforge\fmlcore\1.20.1-47.1.3\fmlcore-1.20.1-47.1.3.jar is missing mods.toml file
[05Oct2023 14:58:45.861] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModFileParser/LOADING]: Mod file C:\Users\chump\AppData\Roaming\com.modrinth.theseus\meta\libraries\net\minecraftforge\javafmllanguage\1.20.1-47.1.3\javafmllanguage-1.20.1-47.1.3.jar is missing mods.toml file
[05Oct2023 14:58:45.877] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModFileParser/LOADING]: Mod file C:\Users\chump\AppData\Roaming\com.modrinth.theseus\meta\libraries\net\minecraftforge\lowcodelanguage\1.20.1-47.1.3\lowcodelanguage-1.20.1-47.1.3.jar is missing mods.toml file
[05Oct2023 14:58:45.889] [main/WARN] [net.minecraftforge.fml.loading.moddiscovery.ModFileParser/LOADING]: Mod file C:\Users\chump\AppData\Roaming\com.modrinth.theseus\meta\libraries\net\minecraftforge\mclanguage\1.20.1-47.1.3\mclanguage-1.20.1-47.1.3.jar is missing mods.toml file
[05Oct2023 14:58:46.784] [main/WARN] [net.minecraftforge.jarjar.selection.JarSelector/]: Attempted to select two dependency jars from JarJar which have the same identification: Mod File:  and Mod File: . Using Mod File: 
[05Oct2023 14:58:46.785] [main/WARN] [net.minecraftforge.jarjar.selection.JarSelector/]: Attempted to select two dependency jars from JarJar which have the same identification: Mod File:  and Mod File: . Using Mod File: 
[05Oct2023 14:58:46.792] [main/INFO] [net.minecraftforge.fml.loading.moddiscovery.JarInJarDependencyLocator/]: Found 12 dependencies adding them to mods collection
[05Oct2023 14:58:56.475] [main/ERROR] [mixin/]: Mixin config dynamiclightsreforged.mixins.json does not specify "minVersion" property
[05Oct2023 14:58:56.491] [main/INFO] [mixin/]: Compatibility level set to JAVA_17
[05Oct2023 14:58:56.544] [main/ERROR] [mixin/]: Mixin config mixins.eco.json does not specify "minVersion" property
[05Oct2023 14:58:57.110] [main/INFO] [mixin/]: Successfully loaded Mixin Connector [ca.spottedleaf.starlight.mixin.MixinConnector]
[05Oct2023 14:58:57.111] [main/INFO] [cpw.mods.modlauncher.LaunchServiceHandler/MODLAUNCHER]: Launching target 'forgeclient' with arguments [--version, 1.20.1, --gameDir, C:\Users\chump\AppData\Roaming\com.modrinth.theseus\profiles\Revitalized, --assetsDir, C:\Users\chump\AppData\Roaming\com.modrinth.theseus\meta\assets, --uuid, 989d88f0eb7e436fb0dad72b97fa4079, --username, _Sketano, --assetIndex, 5, --accessToken, ????????, --clientId, c4502edb-87c6-40cb-b595-64a280cf8906, --xuid, 0, --userType, msa, --versionType, release, --width, 854, --height, 480]
[05Oct2023 14:58:57.483] [main/INFO] [ModernFix/]: Loaded configuration file for ModernFix 5.7.5+mc1.20.1: 63 options available, 1 override(s) found
[05Oct2023 14:58:57.487] [main/WARN] [ModernFix/]: Option 'mixin.perf.thread_priorities' overriden (by mods [smoothboot]) to 'false'
[05Oct2023 14:58:57.489] [main/INFO] [ModernFix/]: Applying Nashorn fix
[05Oct2023 14:58:57.549] [main/INFO] [ModernFix/]: Applied Forge config corruption patch
[05Oct2023 14:58:57.607] [main/INFO] [fpsreducer/]: OptiFine was NOT detected.
[05Oct2023 14:58:57.611] [main/INFO] [fpsreducer/]: OptiFabric was NOT detected.
[05Oct2023 14:58:57.656] [main/INFO] [Embeddium/]: Loaded configuration file for Embeddium: 41 options available, 0 override(s) found
[05Oct2023 14:58:57.900] [main/WARN] [mixin/]: Reference map 'respawnobelisks-forge-refmap.json' for respawnobelisks.mixins.json could not be read. If this is a development environment you can ignore this message
[05Oct2023 14:58:58.035] [main/INFO] [Canary/]: Loaded configuration file for Canary: 123 options available, 0 override(s) found
[05Oct2023 14:58:58.106] [main/WARN] [mixin/]: Reference map 'nucleus-forge-refmap.json' for nucleus.mixins.json could not be read. If this is a development environment you can ignore this message
[05Oct2023 14:58:58.260] [main/INFO] [Rubidium Extra/]: Loaded configuration file for Sodium Extra: 33 options available, 0 override(s) found
[05Oct2023 14:58:58.430] [main/INFO] [Essential Logger - Plugin/]: Starting Essential v1.2.2.2 (#4e34fe63b) [stable]
[05Oct2023 14:59:00.329] [main/INFO] [net.minecraftforge.coremod.CoreMod.placebo/COREMODLOG]: Patching IForgeItemStack#getEnchantmentLevel
[05Oct2023 14:59:00.374] [main/INFO] [net.minecraftforge.coremod.CoreMod.placebo/COREMODLOG]: Patching IForgeItemStack#getEnchantmentLevel
[05Oct2023 14:59:00.836] [main/WARN] [mixin/]: Error loading class: mezz/modnametooltip/TooltipEventHandler (java.lang.ClassNotFoundException: mezz.modnametooltip.TooltipEventHandler)
[05Oct2023 14:59:00.843] [main/WARN] [mixin/]: Error loading class: me/shedaniel/rei/impl/client/ClientHelperImpl (java.lang.ClassNotFoundException: me.shedaniel.rei.impl.client.ClientHelperImpl)
[05Oct2023 14:59:02.808] [main/INFO] [fpsreducer/]: bre2el.fpsreducer.mixin.RenderSystemMixin will be applied.
[05Oct2023 14:59:02.809] [main/INFO] [fpsreducer/]: bre2el.fpsreducer.mixin.WindowMixin will NOT be applied because OptiFine was NOT detected.
[05Oct2023 14:59:07.733] [main/INFO] [MixinExtras|Service/]: Initializing MixinExtras via com.llamalad7.mixinextras.service.MixinExtrasServiceImpl(version=0.2.0-rc.5).
[05Oct2023 14:59:11.510] [main/WARN] [mixin/]: Static binding violation: PRIVATE @Overwrite method m_216202_ in modernfix-forge.mixins.json:perf.tag_id_caching.TagOrElementLocationMixin cannot reduce visibiliy of PUBLIC target method, visibility will be upgraded.
[05Oct2023 14:59:13.621] [main/WARN] [oshi.driver.windows.perfmon.PerfmonDisabled/]: Invalid registry value type detected for PerfOS counters. Should be REG_DWORD. Ignoring: HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\PerfOS\Performance\Disable Performance Counters.
[05Oct2023 14:59:14.110] [pool-4-thread-1/INFO] [net.minecraft.server.Bootstrap/]: ModernFix reached bootstrap stage (46.61 s after launch)
[05Oct2023 14:59:14.899] [pool-4-thread-1/INFO] [ModernFix/]: Injecting BlockStateBase cache population hook into getNeighborPathNodeType from com.abdelaziz.canary.mixin.ai.pathing.BlockStateBaseMixin
[05Oct2023 14:59:14.902] [pool-4-thread-1/INFO] [ModernFix/]: Injecting BlockStateBase cache population hook into getPathNodeType from com.abdelaziz.canary.mixin.ai.pathing.BlockStateBaseMixin
[05Oct2023 14:59:14.903] [pool-4-thread-1/INFO] [ModernFix/]: Injecting BlockStateBase cache population hook into getOpacityIfCached from ca.spottedleaf.starlight.mixin.common.blockstate.BlockStateBaseMixin
[05Oct2023 14:59:14.903] [pool-4-thread-1/INFO] [ModernFix/]: Injecting BlockStateBase cache population hook into isConditionallyFullOpaque from ca.spottedleaf.starlight.mixin.common.blockstate.BlockStateBaseMixin
[05Oct2023 14:59:14.904] [pool-4-thread-1/INFO] [ModernFix/]: Injecting BlockStateBase cache population hook into getAllFlags from com.abdelaziz.canary.mixin.util.block_tracking.BlockStateBaseMixin
[05Oct2023 14:59:18.679] [pool-4-thread-1/INFO] [net.minecraftforge.coremod.CoreMod.placebo/COREMODLOG]: Patching IForgeItemStack#getEnchantmentLevel
[05Oct2023 14:59:18.689] [pool-4-thread-1/INFO] [net.minecraftforge.coremod.CoreMod.placebo/COREMODLOG]: Patching IForgeItemStack#getEnchantmentLevel
[05Oct2023 14:59:24.841] [pool-4-thread-1/INFO] [net.minecraft.server.Bootstrap/]: Vanilla bootstrap took 10713 milliseconds
[05Oct2023 14:59:29.756] [pool-4-thread-1/INFO] [net.minecraftforge.coremod.CoreMod.placebo/COREMODLOG]: Patching IForgeItemStack#getEnchantmentLevel
[05Oct2023 14:59:29.765] [pool-4-thread-1/INFO] [net.minecraftforge.coremod.CoreMod.placebo/COREMODLOG]: Patching IForgeItemStack#getEnchantmentLevel
[05Oct2023 14:59:30.848] [pool-4-thread-1/WARN] [mixin/]: @Final field f_31427_:[I in epicfight.mixins.json:MixinWitherBoss should be final
[05Oct2023 14:59:30.849] [pool-4-thread-1/WARN] [mixin/]: @Final field f_31428_:[I in epicfight.mixins.json:MixinWitherBoss should be final
[05Oct2023 14:59:47.898] [Client Shutdown Thread/ERROR] [net.minecraft.client.main.Main/]: Caught previously unhandled exception :
org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:250) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4]
	at org.spongepowered.asm.service.modlauncher.MixinTransformationHandler.processClass(MixinTransformationHandler.java:131) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4]
	at org.spongepowered.asm.launch.MixinLaunchPluginLegacy.processClass(MixinLaunchPluginLegacy.java:131) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4]
	at cpw.mods.modlauncher.serviceapi.ILaunchPluginService.processClassWithFlags(ILaunchPluginService.java:156) ~[modlauncher-10.0.9.jar:10.0.9+10.0.9+main.dcd20f30]
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:88) ~[modlauncher-10.0.9.jar:?]
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-10.0.9.jar:?]
	at cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:50) ~[modlauncher-10.0.9.jar:?]
	at cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:113) ~[securejarhandler-2.1.10.jar:?]
	at cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219) ~[securejarhandler-2.1.10.jar:?]
	at cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229) ~[securejarhandler-2.1.10.jar:?]
	at cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219) ~[securejarhandler-2.1.10.jar:?]
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135) ~[securejarhandler-2.1.10.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:?]
	at net.minecraft.client.main.Main$2.run(Main.java:164) ~[client-1.20.1-20230612.114412-srg.jar%23314!/:?]
Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: Critical injection failure: Callback method autoHud$wrapTooltip(Lnet/minecraft/client/gui/Gui;Lnet/minecraft/client/gui/GuiGraphics;ILcom/llamalad7/mixinextras/injector/wrapoperation/Operation;)V in autohud-forge.mixins.json:gui.InGameHudMixin failed injection check, (0/1) succeeded. Scanned 1 target(s). Using refmap autohud-forge-refmap.json
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.postInject(InjectionInfo.java:468) ~[mixin-0.8.5.jar%23100!/:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4]
	at com.llamalad7.mixinextras.injector.wrapoperation.WrapOperationInjectionInfo.latePostInject(WrapOperationInjectionInfo.java:48) ~[MixinExtras-0.2.0-rc.5.jar%23353!/:?]
	at com.llamalad7.mixinextras.injector.wrapoperation.WrapOperationApplicatorExtension.postApply(WrapOperationApplicatorExtension.java:39) ~[MixinExtras-0.2.0-rc.5.jar%23353!/:?]
	at org.spongepowered.asm.mixin.transformer.ext.Extensions.postApply(Extensions.java:167) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.postApply(TargetClassContext.java:394) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:367) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4]
	... 14 more
```
I have no idea whats causing it

**Comment by Crendgrim** [comment]
Ugh, I bet Forge changed things around yet again. Thanks for the report.

**Comment by Crendgrim** [comment]
Should be fixed (and for good this time!) with AutoHUD 6.3+forge
